documentation/book/assembly-kafka-bridge-configuration.adoc

```diff
@@ -0,0 +1,31 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-bridge.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-bridge-configuration-{context}']
+
+= Kafka Bridge configuration
+
+{ProductName} allows you to customize the configuration of Apache Kafka Bridge nodes by editing certain options listed in {ApacheKafkaConsumerConfig} and {ApacheKafkaProducerConfig}.
+
+Configuration options that can be configured relate to:
+
+* Kafka cluster bootstrap address
+* Security (Encryption, Authentication, and Authorization)
+* Consumer configuration
+* Producer configuration
+* HTTP configuration
+
+These options are automatically configured by {ProductName}.
```

Comment:
In other parts of the guide it is options that cannot be configured that are automatically configured.